mypy_django_plugin/transformers/storage.py

```diff
@@ -0,0 +1,72 @@
+from mypy.checker import TypeChecker
+from mypy.plugin import AnalyzeTypeContext, MethodContext
+from mypy.semanal import SemanticAnalyzer
+from mypy.typeanal import TypeAnalyser
+from mypy.types import Instance, PlaceholderType, UninhabitedType, get_proper_type
+from mypy.types import Type as MypyType
+from mypy.typevars import fill_typevars
+
+from mypy_django_plugin.django.context import DjangoContext
+from mypy_django_plugin.lib import helpers
+
+
+def get_storage_backend(alias: str, django_context: DjangoContext) -> str | None:
+    "Defensively look for a settings.STORAGES by its alias."
+
+    try:
+        fullname = django_context.settings.STORAGES[alias]["BACKEND"]
+        assert isinstance(fullname, str) and "." in fullname
```

Comment:
Please, convert an `assert` to `if`, since we catch `AssertionError`

Reply:
I am also catching assertion error. I figured this was better than `if check: raise ValueError`, but do you not agree?

Reply:
```python
if not isinstance(fullname, str) or "." not in fullname:
    return None
```

?

Reply:
I forgot that this code was no longer doing anything in the exception handler. It was previously reporting a failure and that's why this code looked like this. I can adjust.

Reply:
I mostly ask, because `assert` can be removed from code with `-OO`, it is fine for sanity-checks and type-narrowing, but I don't think that we can base logics on it :)